src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js

```diff
@@ -264,16 +262,55 @@ foam.CLASS({
 
 
         // Update lifted choices based on their availability
-        let newSelectedData = [...this.selectedData];
+        let newSelectedData = [...this.data.selectedData];
         for ( const w of liftedWizardlets ) {
           if ( w.isAvailable ) newSelectedData.push(w.capability);
           else foam.Array.remove(newSelectedData, w.capability);
         }
-        this.selectedData = foam.Array.unique(newSelectedData);
+        this.data.selectedData = foam.Array.unique(newSelectedData);
       }
       const slots = liftedWizardlets.map(w => w.isAvailable$);
       this.ArraySlot.create({ slots }).sub(updated);
       this.isAvailable$.sub(updated);
-    }
+    },
+    function getPrerequisiteAvailabilitySlot(prereqWizardlet){
+      var slot = foam.core.SimpleSlot.create();
+
+      this.data.selectedData$.sub(
+        ()=> slot.set(this.isPrereqSelected(prereqWizardlet))
+      );
+
+      slot.set(this.isPrereqSelected(prereqWizardlet));
+      
+      return slot;
+    },
+
+    function isPrereqSelected(prereqWizardlet){
+      for ( var i = 0 ; i < this.data.selectedData.length ; i++ ) {
+        var currentData = this.data.selectedData[i];
+
+        let idToCompare;
+        if ( typeof currentData === 'string' ){
```

Comment:
Better to use foam.String.isInstance(currentData)

Reply:
Added.